internal/build/view.go

```diff
@@ -16,8 +16,8 @@ func BuildSummary(build *buildkite.Build) string {
 		buildCreator(build),
 		build.CreatedAt.UTC().Format(time.RFC1123Z))
 	hash := *build.Commit
-	if len(hash) >= 7 {
```

Comment:
Ohh could hash, or rather `build.Commit` actually be null even? So should check for that as well? I guess technically it could 🤷‍♂️ so might be best to check regardless

Reply:
@jradtilbrook that isn't a change in this pr

Reply:
And Go will just return an empty string